<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tbData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>791</width>
      <height>381</height>
     </rect>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btExtract">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btTransform">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>450</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Transform</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btDelete">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>450</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-10</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ETL TOOL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btBrowser">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browser</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEtext">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew_window_2"/>
    <addaction name="actionImport_File"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_window">
   <property name="text">
    <string>New window</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNew_window_2">
   <property name="text">
    <string>New window</string>
   </property>
  </action>
  <action name="actionImport_File">
   <property name="text">
    <string>Import File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
